### ISSUE TITLE
debian desktop client doesn't start due to missing /opt/rocketchat-desktop

### ISSUE THREAD

**Issue by harridu** [RocketChat/Rocket.Chat.Electron#2082]
## My Setup

- Operating System: Debian Sid
- App Version: 3.2.2
- Installation type: dpkg

- [ x] I have tested with the latest version
- [x ] I can simulate the issue easily

## Description
Apparently the Debian package still makes use of /opt/rocketchat-desktop, even though this directory is not provided by the 3.2.2 package anymore. RC doesn't start. Creating a symlink seems to be sufficient as a workaround.

Next I had to run `chmod 4755 /opt/Rocket.Chat/chrome-sandbox`. This could have been included in the Debian package as well.
```
ln -s Rocket.Chat /opt/rocketchat-desktop
chmod 4755 /opt/Rocket.Chat/chrome-sandbox
```

**Comment by tmartincpp** [comment]
Duplicate of #2044.

**Comment by tmartincpp** [comment]
This issue can be closed, it has been fixed in 3.4.0.

**Comment by jeanfbrito** [comment]
Please test if the problem persists with our latest release. I will be closing the issue but if there is any problem yet, we will open it again.
